airflow/dag_processing/manager.py

```diff
@@ -849,7 +849,7 @@ def _log_file_processing_stats(self, known_file_paths):
             rows.append((file_path, processor_pid, runtime, num_dags, num_errors, last_runtime, last_run))
 
         # Sort by longest last runtime. (Can't sort None values in python3)
-        rows.sort(key=lambda x: x[3] or 0.0)
+        rows.sort(key=lambda x: x[5] or 0.0)
```

Comment:
```suggestion
        rows.sort(key=lambda x: x[5] or 0.0, reverse=True)
```

If we want to order in descending order